orangecontrib/text/corpus.py

```diff
@@ -173,6 +173,14 @@ def titles(self):
         """ Returns a list of titles. """
         attrs = [attr for attr in chain(self.domain.variables, self.domain.metas)
                  if attr.attributes.get('title', False)]
+        # Alternatively, use heuristics
+        if not attrs:
+            for var in sorted(self.domain.metas,
```

Comment:
Any specific reason for metas only? `Friends-transcripts` have title as a discrete feature in X.